_source/_posts/2020-12-09-dotnet-cloud-host-publish.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: blog_post
+title: How to Deploy Your .NET Core App to Google Cloud, AWS or Azure
+author: greg-sinka
+by: contractor
+communities: [.net]
+description: "How to deploy a .NET Core app to AWS, Azure or Google Cloud."
+tags: [google-cloud, dotnet, aspnet, csharp, dotnetcore, aspnetcore, aws, azure, deployment]
+tweets:
+- "Learn how to deploy an Okta #dotnetcore app to #AWS, #GoogleCloud or #Azure."
+- "Discover how to deploy and configure a secured #dotnetcore app to #GoogleCloud, #Azure or #AWS!"
+- "Need to deploy your #dotnetcore to a major cloud host like #Azure #AWS or #GoogleCloud? Here you go!"
+image: blog/featured/okta-dotnet-mouse-down.jpg
+type: conversion
+---
+
+There has been a cut-throat competition between cloud hosts in the past few years - each attempting to earn the sympathy of developers and dev-ops by rolling out shiny new tools, plugins, and integrations. There are a gazillion how-to tutorials and guides in the community on using these tools. Sometimes when looking for a solution, it is hard to find the newest and simplest way. I never know if an article written last year is still valid using today's tools. In this article, I'm going to walk you through the deployment of a .NET Core 3.1 application to the major cloud hosts (Microsoft Azure App Service, AWS Beanstalk, Google Cloud App Engine) in the simplest possible way. After deployment, I'll take a few steps forward and configure Okta Authentication in the example application to show how to add a login flow at once.
+
+## What is Okta
```

Comment:
We try not to use Okta in headers, but if you must, I'd add a question mark.

Reply:
You can't add a question mark though

Reply:
Why not? I'm confused.

Reply:
MD026/no-trailing-punctuation: Trailing punctuation in heading [Punctuation: '?'] - markdownlint inside of VS Code

Reply:
I just rewrote it instead

Reply:
That's a bug in VS Code then. My post from Monday has an exclamation point in its first header. :) https://developer.okta.com/blog/2020/12/07/spring-cloud-config